### DISCUSSION TITLE
Open same PWA window when clicking other link to the same PWA

### DISCUSSION THREAD

**Discussion by sapphired18** [filips123/PWAsForFirefox#192] [Ideas]
My idea is, when I click a YouTube link then open other YouTube link. Open it to same YouTube PWA rather than open multiple window

Screenshot
![image](https://user-images.githubusercontent.com/58063311/182592495-0757d272-69f1-43ac-ad5c-e1dafbdd9779.png)
My YouTube profile settings
![image](https://user-images.githubusercontent.com/58063311/182592800-deab711e-cb1c-4cfb-abad-7aaee1c69227.png)

**Comment by filips123** [comment]
It works fine for me, with the same settings as yours. Clicking on a YouTube video (or other link) opens it in a current tab, ctrl-clicking opens it in a new (unfocused) tab, and shift-clicking opens it in a new window. If you've just changed "When opening a link that should..." preference, you should restart web app to apply it. Otherwise, I don't know why it doesn't work for you.

**Reply by sapphired18** [reply]
I forgot to mention I open the link from discord
it does not work for me because discord opens Firefox first then PWA for Firefox redirects it

**Reply by sapphired18** [reply]
https://user-images.githubusercontent.com/58063311/182773347-7f235540-2e66-44ac-8415-92720921da08.mp4

**Reply by sapphired18** [reply]
also not working from any site

**Comment by sapphired18** [comment]
You might think I have opened another youtube link from the YouTube PWA itself. No, I don't mean like that. What I mean is opening it from an external site or Firefox home screen itself. Or I might be wrong thinking you mean this?? Sorry


my English is so bad so I constantly changing the sentences

**Reply by filips123** [reply]
Ok, I understand it now. I will check if I can implement this.

**Reply by sapphired18** [reply]
thank you
